src/coreclr/tools/ILCompiler-DependencyGraph-Viewer/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file is probably not needed now that this is a .NET core app.

Reply:
Looks to not be required. I've removed it.